### ISSUE TITLE
Créer une page tag par langue 

### ISSUE THREAD

**Issue by Aliocha-Iordanoff** [cafebabel/cafebabel.com#451]
Exemple d'URL : https://cafebabel.com/fr/article/tag/creative/

Problème : dans une logique exploratoire, Google va chercher ce qui se trouve dans 
> https://cafebabel.com/fr/article/tag/
> https://cafebabel.com/fr/article/
> https://cafebabel.com/fr/

Or les deux premiers cas renvoient une erreur 404.

Piste de solution : envisager de supprimer le /article/

Ce dossier logique dans les URL pose par ailleurs un autre problème du point de vue du référencement et l'UX, c'est qu'il n'est pas traduit dans les différentes langues.

**Comment by Aliocha-Iordanoff** [comment]
Ce problème est peut-être structurel.
Question à traiter en urgence pour le référencement naturel.

**Comment by davidbgk** [comment]
> dans une logique exploratoire, Google va chercher ce qui se trouve dans

Quelles sont les références menant à cette conclusion ?

**Comment by Aliocha-Iordanoff** [comment]
@davidbgk Je dis cela par expérience, et il n'y a pas de littérature sur le sujet. C'est une approche logique pour l'internaute, donc pour Google. 
Ce n'est pas du tout un point bloquant pour le référencement naturel. C'est juste une structuration de l'information que le moteur de recherche pourra utiliser si toutefois il y a une cohérence arborescente dans ces dossiers et sous-dossiers. Sinon, c'est simplement considéré comme une simple chaine de caractères.
cc @matthieuamare 

Il n'y a donc pas forcément de caractère prioritaire pour modifier l'existant. Par contre, cela prend un caractère urgent si vous décidez de le faire, car dans l'hypothèse où vous changeriez de structure d'URL et de sous-dossier à l'avenir, cela impliquerait une nouvelle modification des URL. Ce serait dommage.

**Comment by matthieuamare** [comment]
À lier avec #464 ? La solution ne serait-ele pas de supprimer "/tag" et "/article ?"

**Comment by Aliocha-Iordanoff** [comment]
Les articles ne sont pas placés dans des sous-dossiers. Donc ce sont deux choses différentes/

le /tag/ permet de comprendre que la suite de l'URL est un tag, et le mot est très court. Pas besoin de supprimer ce noeud logique.
Par contre, dans cette url https://cafebabel.com/fr/article/tag/creative/ , 
le **/article/** pourrait être supprimé oui.

**Comment by ffameli** [comment]
créer une page .fr/tag dans toutes les langues : nécessite un temps de dév

**Comment by vinyll** [comment]
En imaginant qu'on supprime les dossiers, mais qu'on conserve les ID, on aurait une structure du type:

https://cafebabel.com/fr/vita-bella-12345566778
https://cafebabel.com/fr/nouveautes-paris-12345566778
https://cafebabel.com/fr/patrice-ramier-12345566778

Du coup difficile de distinguer ce qu'est un utilisateur, un tag, un article pour un utilisateur, comme pour le système (et alourdissement des performances).

Une proposition de @davidbgk est d'abbréger le prefix dans l'URL :
https://cafebabel.com/fr/t/vita-bella
https://cafebabel.com/fr/a/nouveautes-paris-12345566778
https://cafebabel.com/fr/p/patrice-ramier-7645345468

`/t` -> tag
`/a` -> article
`/p` -> profil

**Comment by ffameli** [comment]
- Créer une page tag par langue afin de rendre existant la page cafebabel.com/fr/tag
cette page recenserait tout simplement les différents tags présents sur le site
voir si cela suffit pour garder des url propres du style : cafebabel.fr/tag/creative/bal-bla-ID1234

**Comment by ffameli** [comment]
Nous pourrions garder les termes /tag et /profil 
Problème de langue pour profil --> solution a : traduire profil en 6 langues et choisir un seul sous dossier langue pour le profil de l'auteur ; solution b : couper le mot profil pour ne pas choisir où ranger le contributeur

**Comment by Aliocha-Iordanoff** [comment]
Du point de vue du confort pour l'internaute, il est préférable d'avoir un mot entier plutôt qu'une seule lettre, qui ne sera pas compréhensible. Par ailleurs cela n'apporterait aucun bénéfice pour le référencement naturel.

**Comment by ffameli** [comment]
renvoyer vers la home pourrait être suffisant pour la page tag

**Comment by Aliocha-Iordanoff** [comment]
Non, ce n'est pas utile pour lé référencement de mettre une redirection ver la homepage. @ffameli

**Comment by matthieuamare** [comment]
Rester sur l'idée de créer une page qui list le titre, la description et potentiellement le picto du tag. Johan s'occupera d'habiller correctement ces pages.

**Comment by ffameli** [comment]
pas forcément utile 
voici des exemples de site/média où les pages intermédiaires sont des erreurs : 
http://www.liberation.fr/france/2018 --> erreur
http://www.liberation.fr/politiques/2018/05/14/vaffanculo_1649971 --> article visible
https://steemit.com/travel/@vcelier/makis-yasaka-jinja-shrine-and-maruyama-park-by-japon-tranlated-from-french
https://steemit.com/travel/@vcelier/

**Comment by vinyll** [comment]
D'autres exemples au hasard qui ne gèrent pas des pages intermédiaires par modifications d'url :
- https://www.atom.io/api/packages/python-jedi/versions/0.3.8/tarball
- https://github.com/cafebabel/cafebabel.com/tree/header-arrangements

Du coup je sais pas quoi trop en penser 🤔

**Comment by Aliocha-Iordanoff** [comment]
Je sais bien que c'est très courant et même majoritaire sur le web. ça n'en fait pas une bonne pratique pour autant !
Les constructions d'URL de type ndd.com/année/mois/jour/xxx.html ont aussi une autre impact, parfois souhaité, c'est que les articles sont très vites considérés comme obsolète. Les stratégie de référencement naturel des média d'actualité sont très particulières, car ils n'ont pas toujours intérêt à ce que de vieux articles remontent bien dans les résultats en raison du fait qu'ils sont très vite obsolètes.

Si c'est compliqué de rendre ces nœuds d'arborescence consultables, ne le faite pas. Cela n'empêchera pas Google de référencer les pages de bas d'arborescence.
